engines/dradis-api/db/migrate/20150722125412_add_api_token_to_users.rb

```diff
@@ -0,0 +1,21 @@
+class AddAPITokenToUsers < ActiveRecord::Migration[5.1]
+  # Future-proof this migration by ignoring everything in the actual model.
```

Comment:
This is neat but is it the way we normally handle this situation?

Reply:
Updated to modern times ✅